ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -63,25 +64,45 @@ - (id)initWithDataSource:(id<ADTokenCacheDataSource>)dataSource
     return _dataSource;
 }
 
-- (ADTokenCacheItem *)getItemForUser:(ADUserIdentifier *)identifier
+- (ADTokenCacheItem *)getItemForUser:(NSString *)userId
```

Comment:
just curious: why was this change necessary?

Reply:
It was a change I made because on the write side we have a userId not an identifier. This PR is from changes I pulled out to focus on read.